components/desktop/mate/mate-calc/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Till Wegmueller
+#
+
+include ../../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		mate-calc
+COMPONENT_VERSION=	1.8.0
+COMPONENT_PROJECT_URL=	http://www.mate-desktop.org
+COMPONENT_SUMMARY=	MATE Calculator App
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:13d56287430cbb7d60a74666ba752a1a8ea9692125c6a02f97d0cc98d451d99a
+COMPONENT_ARCHIVE_URL=	http://pub.mate-desktop.org/sources/$(COMPONENT_NAME)/1.8/$(COMPONENT_ARCHIVE)
+COMPONENT_FMRI=		desktop/mate/$(COMPONENT_NAME)
```

Comment:
In that case lets not make a complete mess and keep all components of a project under similar FMRI. eg. library/desktop/mate desktop/mate which also has some base in the file structure of oi-userland. Thus giving contributors some clue where to look for when updating components.